packages/vuetify/src/components/VSpeedDial/VSpeedDial.js

```diff
@@ -65,7 +65,7 @@ export default {
     if (this.isActive) {
       let btnCount = 0
       children = (this.$slots.default || []).map((b, i) => {
-        if (b.tag && b.componentOptions.Ctor.options.name === 'v-btn') {
+        if (b.tag && typeof b.componentOptions !== 'undefined' && b.componentOptions.Ctor.options.name === 'v-btn') {
```

Comment:
```suggestion
        if (b.tag && typeof b.componentOptions && b.componentOptions.Ctor.options.name === 'v-btn') {
```

Reply:
Should probably be any non-empty vnode, dunno why it singles out buttons.

Reply:
@johnleider is this typescript specific?
in javascript `typeof something` results in a string `'undefined'`
And then it would always results in true because it's a string.

Anyway the code you suggested results in the same error.

Reply:
@KaelWD Could you tell me how to do that? Your solution sounds better, if i understand it correctly you mean that we change the if to check if there is any vuetify component (starts with `v-`)

Perhaps we should ask @kuromoka and @sh7dm if there was any particular reason for this

Reply:
My suggested change was merely for style purposes. As far as @KaelWD 's suggestion. This is used in a few places, maybe it's time to normalize the functionality into a helper of sorts.

Reply:
Hi John,

But when i apply your auggestion and test it, it breaks.

Reply:
> check if there is any vuetify component (starts with v-)

any element, not just vuetify components

Reply:
How could i get this PR accepted?
there is a lot of discussion and good suggestions, but nothing specific.

John's comment breaks code and i don't get what is better about it since only `!== 'undefined'` is removed.

Kael's comment is a really good suggestion, but i don't know how to implement that.

Reply:
I'm facing same issue and code change above fixes it

Reply:
My suggestion was just to change the compare logic. If it doesn't work, so be it.

As far as getting this accepted, I'd say that this works and let's get it in.